src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -1703,26 +1703,22 @@ End Module").Path
         End Sub
 
         <Fact>
-        Public Sub LanguageVersion_CommandLineUsage()
-            Dim expected = [Enum].GetValues(GetType(LanguageVersion)).Cast(Of LanguageVersion)().Select(Function(v) v.ToDisplayString())
-            Dim help = VBResources.IDS_VBCHelp
-
-            Dim rangeStart = help.IndexOf("/langversion")
-            Dim rangeEnd = help.IndexOf("/optionexplicit")
-            Assert.True(rangeEnd > rangeStart)
+        Public Sub LanguageVersion_ListLangVersions()
+            Dim dir = Temp.CreateDirectory()
+            Dim outWriter As New StringWriter()
+            Dim exitCode As Integer = New MockVisualBasicCompiler(Nothing, dir.ToString(), {"/langversion:?"}).Run(outWriter, Nothing)
+            Assert.Equal(0, exitCode)
 
-            Dim helpRange = help.Substring(rangeStart, rangeEnd - rangeStart).ToLowerInvariant()
-            Dim acceptableSurroundingChar = {CChar(vbCr), CChar(vbLf), "|"c, " "c}
+            Dim actual = outWriter.ToString()
+            Dim expected = [Enum].GetValues(GetType(LanguageVersion)).Cast(Of LanguageVersion)().Select(Function(v) v.ToDisplayString())
+            Dim acceptableSurroundingChar = {CChar(vbCr), CChar(vbLf), "("c, ")"c, " "c}
```

Comment:
>"("c, ")"c [](start = 71, length = 10)

I think we would want to remove these  surrounding chars, or test that ```default``` and ```latest``` appear twice.